datumaro/components/environment.py

```diff
@@ -322,5 +354,11 @@ def _register(registry: PluginRegistry):
 
         return merged
 
+    @classmethod
+    @contextmanager
+    def release_builtin_plugins(cls):
```

Comment:
This method seems no use to users. Why do we have to implement it in `Environment` rather than a stand alone in test file? Would you give some examples?

Reply:
I wanted to open it for users who want to reload built-in plugins somehow.